guides/plugins/plugins/framework/data-handling/field-inheritance.md

```diff
@@ -0,0 +1,227 @@
+# Field inheritance
+
+## Overview
+
+In this guide you'll learn how to create inherited fields for your entities.
+
+## Prerequisites
+
+This guide is built upon the [Plugin Base Guide](../../plugin-base-guide.md), but any plugin will work here. Just note that all examples are using the plugin mentioned above.
+
+You also should have a look at our [Adding custom complex data](./add-custom-complex-data.md) guide, since this guide is built upon it.
+
```

Comment:
I'm missing an Intro inheritance at all.
Is there another guide, I did not find ? 
You explain, how to, but not what field inheritance is.